x-pack/platform/plugins/private/reporting/public/redirect/redirect_app.tsx

```diff
@@ -83,6 +89,7 @@ export const RedirectApp: FunctionComponent<Props> = ({ apiClient, screenshotMod
           );
         }
 
+        core.chrome.setIsVisible(false);
```

Comment:
This should fix https://github.com/elastic/kibana/issues/227187 for grid layout. We basically hide chrome using JS and this makes sure the grid layout dimensions are updated. 

the css `hide-for-sharing` class doesn't work for grid layout because even though side nav is hidden, the space for it is still reserved by the layout

Reply:
this assumes there is not page reload during this navigation. there shouldn't be